### PR TITLE
List Matches Feature added

### PR DESCRIPTION
**Pull request by Sivasubramanian35**
- [x] List Available Matches Feature is added

### CODE REVIEW

File: src/main/java/in/siva/model/Matches.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package in.siva.model;
+
+import java.time.LocalDate;
+
+public class Matches {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/siva/model/Matches.java]: Rename "Matches" to "Match"

File: src/main/webapp/index.jsp [hunk 2]
```diff
@@ -1,20 +1,13 @@
 <%@page import="java.util.Date"%>
 <%@page import="java.time.LocalDate"%>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp]: Remove unused imports

File: src/main/webapp/matches.jsp [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+				int i = 0;
+				for (Matches match : matchList) {
+					i++;
+				%>
+				<tr>
+					<td><%=i%></td>
+					<td><%=match.getTeam1()%></td>
+					<td><%=match.getTeam2()%></td>
+					<td><%=match.getMatchDate()%></td>
+					<td><a href="">Book</a></td>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/matches.jsp]: Remove booking link

**Review by ghost — CHANGES_REQUESTED** [review]
Added comments

File: src/main/java/in/siva/model/Matches.java (hunk 1 above)
Comment (Sivasubramanian35) [reply src/main/java/in/siva/model/Matches.java]: ok sir

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**